apache-php/files/apache-foreground

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+rm -f /var/run/apache2/apache2.pid
+
+exec /usr/sbin/apache2ctl -DFOREGROUND
```

Comment:
Almost all images does not put a space between them, I didn't know :D, I'll correct